docs/release-files.md

```diff
@@ -0,0 +1,75 @@
+Releasing hsd
+=============
+
+This document contains information about bundling, signing and
+distributing the files.
+
+`hsd` is distributed through several platforms: `github`, `npm`, `brew`.
+
+<!-- markdown-toc -i release-files.md -->
+
+<!-- toc -->
+
+- [Deploying to github (tag)](#deploying-to-github-tag)
+  * [Major, minor and patches](#major-minor-and-patches)
+- [Deploying to npm](#deploying-to-npm)
+  * [Deploying latest version, minor and patches included](#deploying-latest-version-minor-and-patches-included)
+  * [Deploying support versions (previous and life-support)](#deploying-support-versions-previous-and-life-support)
+- [Deploying to homebrew](#deploying-to-homebrew)
+
+<!-- tocstop -->
+
+## Deploying to github (tag)
+
+  This does not need many additional actions as we use github as our primary
+git platform. After each release we need to update INSTALL.md versions. Tags
+MUST to be signed.
+
```

Comment:
maybe move the "note" from line 48 up here and explain exactly what goes in the tagged commit (i.e. just version bumps in package.json and readme, etc)

Reply:
Release file describes existing process, so decided not to expand in this file. I am planning to write new release process (at least attempt) in a different PR.

Reply:
okie doke.